src/spatialindex.jl

```diff
@@ -21,13 +21,10 @@ Base.show(io::IO, ::MIME"text/plain", R::SpatialIndex{M}) where M = print(io, "S
 Base.show(io::IO, ::SpatialIndex{M}) where M = print(io, "SpatialIndex using ", _value(M))
 Base.summary(io::IO, R::SpatialIndex) = show(io::IO, R)
 
-GeoInterface.isgeometry(::Type{<:AcceleratedArray{T,N,A,<:AbstractSpatialUniqueIndex}} where {T,N,A}) = true
-GeoInterface.isgeometry(::Type{<:SpatialIndex}) = true
-GeoInterface.geomtrait(::AcceleratedArray{T,N,A,<:AbstractSpatialUniqueIndex}) where {T,N,A} = GeoInterface.PolygonTrait()
-GeoInterface.geomtrait(::AbstractSpatialUniqueIndex) = GeoInterface.PolygonTrait()
+GeoInterface.isgeometry(::Type{<:AcceleratedArray{T,N,A,<:AbstractSpatialUniqueIndex}} where {T,N,A}) = false
```

Comment:
Removing these two methods completely would have the same result no?

Reply:
True, we could do either